common/netlink.cpp

```diff
@@ -0,0 +1,468 @@
+#include "netlink.h"
+
+#include <linux/if_arp.h> // ARPHRD_ defines
+
+#include <unistd.h> // getpid(), getpagesize()
+#include <string.h> // memcpy, memset etc
+
+Netlink::Netlink()
```

Comment:
What do you think, should we name the class NetworkManager and the CapabilityManager?